### DISCUSSION TITLE
如何只监听ipv4的0.0.0.0, 不监听ipv6的[::] ?

### DISCUSSION THREAD

**Discussion by GitCourser** [AlistGo/alist#7119] [Q&A]
配置里 `scheme.address` 的值设 `0.0.0.0` 还包括 ipv6 的 `[::] `吗, 只想监听ipv4地址 `0.0.0.0` 该怎么做?

**Comment by pongfcnkl** [comment]
关掉设备的ipv6

**Reply by GitCourser** [reply]
ipv6有用, 只是不想这个走ipv6, 除了用防火墙外还有其他办法吗
